tendrl-gluster-integration.spec

```diff
@@ -39,6 +39,7 @@ install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/gluster-integ
 install -Dm 0644 tendrl-gluster-integration.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-gluster-integration.service
 install -Dm 0644 etc/tendrl/gluster-integration/gluster-integration.conf.yaml.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/gluster-integration/gluster-integration.conf.yaml
 install -Dm 644 etc/tendrl/gluster-integration/*.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/gluster-integration/
+install -Dm 0755 tendrl/gluster-integration/sds_sync/vol_utilization.py %{_bindir}/vol_utilization
```

Comment:
This shouldnt be handled here,

youll need to add an entry for "tendrl_gluster_vol_utilizatin: tendrl.gluster-integration.sds_sync/vol_utilization"

https://github.com/Tendrl/gluster-integration/blob/master/setup.py#L110

Reply:
Done. Will update